### DISCUSSION TITLE
Best way to start and manage a Doks project with git

### DISCUSSION THREAD

**Discussion by pensivedog** [thuliteio/doks#838] [Q&A]
Git newbie question: the Doks quick start guide says to use `git clone https://github.com/h-enk/doks.git my-doks-site`. But for the purposes of building a website based on Doks (rather than contributing back to the Doks project), many people might want to start fresh, without the full commit history. The Doks github repo has a "Use this template" button, which I understand is the same as

```
git clone https://github.com/h-enk/doks.git my-doks-site
rm -rf .git
git init
```
But in this case, how do we get upstream updates and improvements from github.com/h-enk/doks?

Grateful for some feedback about approaches and best practices others are using to develop their Doks-based websites.

**Comment by FactoryOptic** [comment]
I believe the Child theme will use the latest, but won't allow any customization. Perhaps look at using this.

The Starter Theme will allow theme changes, but it's up to you to manage bringing in any updates, which makes sense. If you modify the theme, you should manage future changes manually, so you don't overwrite your changes.

**Reply by Natureshadow** [reply]
> The Starter Theme will allow theme changes, but it's up to you to manage bringing in any updates, which makes sense.

Not at all. It is awkward and unnecessary.

Hugo can separate content from the theme for a reason, and this is the only theme I have seen so far that breaks this rule. For no obvious reason, and sadly making it entirely useless for a serious documentation project (sadly, because it is apart from that a great an beautiful theme).
